app/Console/Commands/DevFakeServersCommand.php

```
@@ -0,0 +1,231 @@
+                $this->stopAll($processes);
+
+                return self::FAILURE;
+            }
+
+            $process = new Process(
+                [PHP_BINARY, '-S', "127.0.0.1:{$port}", $router],
+                base_path(),
+                ['FAKE_SERVER_PROFILE' => json_encode($profile)],
+            );
+            $process->setTimeout(null);
+            $process->start(function ($type, $buffer) use ($port, $stream) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$type'. (undefined)

(UnusedFormalParameter)